src/azul/deployment.py

```diff
@@ -91,18 +91,26 @@ def es_endpoint(self) -> Optional[Netloc]:
             return config.es_endpoint
         else:
             self._es_config_warning('endpoint')
-            return self._describe_es_domain_status['Endpoint'], 443
+            es_domain_status = self._describe_es_domain_status
```

Comment:
```suggestion
            status = self._describe_es_domain_status
```

Small scope, short names.